src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py

```diff
@@ -162,6 +163,11 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
                    help='The type of token. Allowed values: {}.'.format(", ".join(get_token_type_completion_list())))
         c.argument('open_id_connect_discovery_document', arg_group='Token Restriction', help='The OpenID connect discovery document.')
         c.argument('widevine_template', arg_group='Widevine Configuration', help='JSON widevine template. Use @{file} to load from a file.')
+        c.argument('ask', arg_group='FairPlay Configuration', help='The key that must be used as FairPlay ASk.')
```

Comment:
It should be 'ASK' for Application Secret Key (https://docs.microsoft.com/en-us/azure/media-services/previous/media-services-protect-hls-with-fairplay)